### DISCUSSION TITLE
Предложение по поводу функции "приглядеться"

### DISCUSSION THREAD

**Discussion by travka4000** [octothorp-team/public-dobrograd#767] [Предложения]
**Думаю у всех была ситуация, что лежит труп и тебе надо для того чтобы отыграть рп и оказать пмп и тд. Нужно понять куда попали, и куда именно, но призрак не всегда отвечает на твой it, так вот идея в том чтобы когда ты приглядывался ты мог заметить куда попали в голову, область груди и тд, были гематомы  синяки, текла кровь? быстро или медленно, короче полная характеристика.**

**Comment by damirmd** [comment]
Ну такое. Упор идёт в механ. Если игрок в виде призрака не хочет отвечать, пусть не отвечает, значит играть не хочет.
Против

**Comment by Kyper1** [comment]
Зачем тебе отыгрывать это самому себе? Если он не отвечает - уходи.

**Reply by travka4000** [reply]
понятно
